src/Microsoft.AspNetCore.Authentication.Abstractions/ISecureDataFormat.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    public interface ISecureDataFormat<TData>
```

Comment:
The Data interfaces are kinda specific to our Security repo implementations, these could live inside of the Security auth package instead...  Thoughts @Tratcher ?